src/gui/README.md

```diff
@@ -644,6 +644,54 @@ To remove all the rulers:
 gui::clear_rulers
 ```
 
+### Add Label to Layout
+
+To add a label to the layout use the following command:
+
+Returns: name of the newly created label.
+
+```tcl
+gui::add_label
+    x y
+    text
+    [color]
+    [size]
+    [name]
+```
+
+#### Options
+
+| Switch Name | Description |
+| ---- | ---- |
+| `x, y,` | point of the label in microns. |
```

Comment:
The center point of the label (I guessed it would be the lower-left).  It might be nice to have an alignment option.